app/crm/api.py

```
@@ -0,0 +1,27 @@
+
+from fastapi import APIRouter, Depends
+
+from app.crm.auth import crm_admin_user
+from app.schemas import UserInfo
+
+router = APIRouter()
+
+
+@router.get("/health")
+async def crm_health(
+    current_user: UserInfo = Depends(crm_admin_user),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)